utils/node-loader/src/batch_pool/batch/upload_code.rs

```diff
@@ -0,0 +1,22 @@
+//! Upload code task.
+
+use crate::{
+    batch_pool::{generators, Seed},
+    utils::LoaderRng,
+};
+
+pub struct UploadCodeArgs(pub Vec<u8>);
+
+impl From<UploadCodeArgs> for Vec<u8> {
+    fn from(UploadCodeArgs(code): UploadCodeArgs) -> Self {
+        code
+    }
+}
+
+impl UploadCodeArgs {
+    pub fn generate<Rng: LoaderRng>(code_seed: Seed) -> Self {
```

Comment:
Btw for args generation may be some trait introduced (comment just for future)

Reply:
will bear in mind